app/assets/src/components/PipelineSampleReport.jsx

```diff
@@ -54,6 +60,8 @@ class PipelineSampleReport extends React.Component {
       { text: "NR %id", value: "NR_percentidentity" },
       { text: "R log(1/e)", value: "NR_neglogevalue" }
     ];
+    this.category_child_parent = { Phage: "Viruses" };
+    this.category_parent_child = { Viruses: ["Phage"] };
```

Comment:
Can this be expressed so that the relationship only needs to be specified once?

Reply:
This a temp hack mostly for performance. I will refactor this once I refactor the rest of the page.